src/App.js

```diff
@@ -60,13 +72,20 @@ export function App () {
         setAppMode(APP_STATE.FILE_VIEW);
     };
 
+    useEffect(() => {
```

Comment:
Let's keep all hooks at the top of the method so that they're easy to find.

Reply:
Done.